doc/security/_index.md

```diff
@@ -0,0 +1,111 @@
+This enables the [Silicon Owner][5] to run signed code on the device.
+
+### Secure Boot
+
+OpenTitan supports a secure boot implementation anchored in the silicon mask ROM.
+The mask ROM contains a set of public keys used to verify the first boot stage stored in flash.
+Isolation between boot stages is enforced by a one way execution model where the current software stage is not allowed to call code in previous stages.
+
+Each boot stage is in charge of verifying the signature of the next stage and locking out portions of the chip that are not required by later stages.
+Once the boot flow reaches kernel execution, the implementation may opt to implement dynamic isolation between applications using the available [Physical Memory Protection][6] (PMP) unit.
+
+### A/B Firmware Updates
+
+OpenTitan supports a firmware layout with two flash partitions, supporting active and non-active instances of each software component.
+This enables a firmware update implementation in which the active partition flashes the new software into the non-active region with minimal downtime.
+Secure boot ensures the integrity and stability of the new software before marking it as active.
+
+Each boot stage is expected to try to launch the newest available version of the next boot stage upon successful signature verification.
+Newest is defined as the software with a more recent build timestamp or larger versioning value.
+
+### Strong Cryptographic Identity
+
+Each OpenTitan silicon device is personalized as part of the manufacturing process with entropy, a unique global identifier, and other values, which are the basis of the chip’s cryptographic identity.
+This identity is referred to as the Creator Identity, and it is fixed throughout the lifetime of the silicon.
+
+The Creator Identity is endorsed by the Silicon Creator and lowRISC via a Transport Creator Certificate which can be used in device authenticity attestation flows.
+
+The Creator Identity is a component of other derived identities that are generated with additional input parameters from the software running on the device, as well as other components provisioned by the Silicon Owner.
+
+### Ownership Transfer
+
+The owner of the silicon is allowed to change in a process known as Ownership Transfer.
+During this process, the existing owner shall wipe any stored secrets before relinquishing ownership of the device.
+At this point a new party can take ownership of the device and provision its own identity.
+This allows the new Silicon Owner to run its own signed code with attestation measurements anchored in the PKI of their choice.
+Attested software can then proceed to provision entropy into the key manager plus additional constants to generate hidden keys strongly bound to the software configuration.
+
+More details for this are available in the [OpenTitan Security Model][7].
+
+
+[1]: https://docs.opentitan.org/doc/security/use_cases/
```

Comment:
These are not `relref`, I assume the only downside is the lack of link checking by `build_docs`? @gkelly

Reply:
Thanks for pointing this out. `relref` seems to be the correct way of defining local refs. Updated the links.

Reply:
Yeah, that's the only real downside is no `build_docs` failure if things move.